File: atcoder_beginners_selection/ABC088B.rb
n = gets.to_i
arr = gets.chomp.split(" ").map(&:to_i)
a_sum = 0
0.step(n-1,2){|i|
	a_sum += arr.sort.reverse[i]
}
b_sum = 0
1.step(n-1,2){|i|
	b_sum += arr.sort.reverse[i]
}

print a_sum - b_sum

File: atcoder_beginners_selection/abc088b.py
n = int(input())
arr = list(map(int,input().split()))
arr.sort(reverse=True)
a = 0
b = 0
for index, num in enumerate(arr):
  if index % 2 == 0:
    a = a + num
  else:
    b = b + num
print(a-b)